File: src/com/dafttech/terra/DesktopStarter.java
package com.dafttech.terra;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;

public class DesktopStarter {

    public static void main(String[] args) {
        new LwjglApplication(new TerraInfinita(), "TerraInfinita", 800, 600, true);
    }

}

File: src/main/scala/com/dafttech/terra/DesktopStarter.scala
package com.dafttech.terra

import com.badlogic.gdx.backends.lwjgl.LwjglApplication

/**
 * Created by LolHens on 18.04.2015.
 */
object DesktopStarter {
  def main(args: Array[String]): Unit = new LwjglApplication(TerraInfinita.$, "TerraInfinita", 1440, 900)
}
